openml/evaluations/functions.py

```diff
@@ -15,10 +15,10 @@ def list_evaluations(
     function: str,
     offset: Optional[int] = None,
     size: Optional[int] = None,
-    id: Optional[List] = None,
     task: Optional[List] = None,
     setup: Optional[List] = None,
     flow: Optional[List] = None,
+    run: Optional[List] = None,
     uploader: Optional[List] = None,
     tag: Optional[str] = None,
```

Comment:
If we're going for more intuitive names, should be change them to be plural, as they expect a list? Or perhaps event better, plural and `_id` suffix (e.g. `run_ids`), to indicate we are looking for something that parses to an id and not actual `OpenMLX` (e.g. `OpenMLRun`) objects instead?

Do we also want to change the typehint to `Optionl[List[Union[str, int]]]`? It's pretty verbose, but more complete... Maybe it would be enough to include this in the docstrings.

Reply:
I'd be happy to change those, but I think this is for a future PR while this is about renaming an argument from `id` to `run`. Shall I open an issue so we don't forget about this?

Reply:
Just did this so I can merge & close PR. See #804.